zipline/pipeline/factors/basic.py

```diff
@@ -54,6 +54,28 @@ def compute(self, today, assets, out, close):
         out[:] = (close[-1] - close[0]) / close[0]
 
 
+class PercentChange(CustomFactor):
+    """
+    Calculates the percent change in the input value over the given window_length.
+    Note: Uses the formula "new - old / abs(old)" to handle negative input values
+
+    **Default Inputs:** None
+    """
+    window_safe = True
+
+    def _validate(self):
+        super(PercentChange, self)._validate()
+        if self.window_length < 2:
+            raise ValueError(
+                "'PercentChange' expected a window length of at least 2, but was "
+                "given {window_length}. For daily percent change, use a window "
```

Comment:
Looks like the [build failed ](https://ci.appveyor.com/project/Quantopian/zipline/builds/26082715/job/i0ylb6hyd414mcxm) due to this line and a couple others being too long. If you run flake8 on this file it will tell you which ones, but it requires a maximum of 80 characters per line. In this case you can just shorten these strings in each line.